### PR TITLE
added nav bar

### PR DESCRIPTION
**Pull request by morecallan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 <html>
 <head>
+<link href="https://fonts.googleapis.com/css?family=Zilla+Slab+Highlight" rel="stylesheet">
 <link rel="stylesheet" type="text/css" href="./main.css">
 	<title>LAUREN</title>
 </head>
 <body>
+	<nav>
+		<ul>
+			<li> <a href="./resue.html">Resume Page</a> </li>
```
Comment (rousell) [inline index.html]: TYPO

Comment (rousell) [inline index.html]: please fix

Comment (morecallan) [inline index.html]: 🥞

Comment (rousell) [inline index.html]: 🍓 🚙 🌴

File: index.html [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+			<li><a href="#">Useless link</a></li>
```
Comment (zoeames) [inline index.html]: why bother putting this in?

**Comment by morecallan** [comment]
@zoeames @rousell I fixed all the shit.

**Comment by rousell** [comment]
@morecallan can you make it lavender while zoe is talking?

**Comment by rousell** [comment]
plz?

**Comment by morecallan** [comment]
👖

**Comment by zoeames** [comment]
👍

**Comment by rousell** [comment]
🎉 👍

**Comment by rousell** [comment]
😢

**Pull request merged**